perf/benchmark/values.yaml

```diff
@@ -15,7 +15,7 @@ appresources1:
 
 fortioImage: fortio/fortio:latest
 curlImage: tutum/curl:trusty
-nighthawkImage: envoyproxy/nighthawk-dev:latest
+nighthawkImage: envoyproxy/nighthawk-dev:59683b759eb8f8bd8cce282795c08f9e2b3313d4
```

Comment:
shouldn''t we point to latest?

Reply:
not yet, we have discussed this with Otto.